menas/ui/service/RuleService.js

```diff
@@ -77,13 +79,12 @@ var RuleService = new function () {
       });
       delete oRule.joinConditions;
     }
-    console.log(JSON.stringify(oRule)); // TODO: remove debug
 
-    Functions.ajax("api/dataset/" + sDatasetName + "/rule/create", "POST", oRule,
+    Functions.ajax("api/dataset/" + oCurrentDataset.name + "/rule/create", "POST", oRule,
```

Comment:
Not sure what dataset name can be, do we need to encode it?

Reply:
Encoding it, might not be a bad idea, any idea if ajax handles encoding automatically?

Reply:
Seems like we should do it by hand. [StackOverflow](https://stackoverflow.com/questions/6544564/url-encode-a-string-in-jquery-for-an-ajax-request)

Reply:
Good idea.